www/galleri/fetchPics.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+outfolder="bilder/pvv-photos"
+folders=$(find /home/pvv -maxdepth 3 -name 'pvv-photos' 2>/dev/null)
```

Comment:
Bør virkelig denne kjøre lokalt og ikke pä microbel? Jeg er redd dette kan gå tregt over nfs, men om du har tested det er alt vell.

Reply:
som en cron jobb har det lite å bety, men hvis det skjer per request er dette ikke bra

Reply:
tror forresten denne ikke finner `/home/pvv/t/{pederbs,yorinad}` ;P

Reply:
Har fungert omtrent like fort på demiurgen som på microbel. Tiltenkt cronjob / systemd timer for eksempel 1-2 ganger i døgnet.